content/zh-cn/docs/contribute/blog/mirroring.md

```diff
@@ -0,0 +1,144 @@
+这些文章与 Kubernetes 有一定的关联。
+
+有些文章会同时出现在两个博客上：一篇文章是主要版本，另一篇是另一个博客上的**镜像文章**。
+
+此页面描述了镜像的标准、镜像的动机，并解释了为确保文章能够在两个博客上都成功发布你需要做些什么。
+
+# {{% heading "prerequisites" %}}
+
+<!--
+Make sure you are familiar with the introduction sections of
+[contributing to Kubernetes blogs](/docs/contribute/blog/), not just to learn about
+the two official blogs and the differences between them, but also to get an overview
+of the process.
+-->
+请确保你已熟悉[为 Kubernetes 博客贡献内容](/zh-cn/docs/contribute/blog/)的介绍部分，
+这不仅是为了了解两个官方博客及其之间的区别，还能帮助你概览整个发布流程。
+
+<!-- content -->
+
+<!--
+## Why we mirror
+
+Mirroring is nearly always from the contributor blog to the main blog. The project does this
+for articles that are about the contributor community, or a part of it, but are also relevant
+to the wider set of readers for Kubernetes' main blog.
+-->
+## 为什么进行镜像
+
+镜像几乎总是从贡献者博客到主博客。项目组会针对那些关于贡献者社区或其部分内容的文章进行镜像，
+但这些文章同时也与 Kubernetes 主博客的更广泛读者相关。
+
+<!--
+As an author (or reviewer), consider the target audience and whether the blog post is appropriate for the [main blog](/docs/contribute/blog/#main-blog).
+For example: if the target audience are Kubernetes contributors only, then the
+[contributor blog](/docs/contribute/blog/#contributor-blog).
+may be more appropriate;
+if the blog post is about open source in general then it may be more suitable on another site outside the Kubernetes project.
+
+This consideration about target audience applies to original and mirrored articles equally.
+-->
+作为作者（或评审者），请考虑目标受众，并判断该博客文章是否适合发布在[主博客](/zh-cn/docs/contribute/blog/#main-blog)上。  
+例如：如果目标受众仅限于 Kubernetes 的贡献者，
+那么[贡献者博客](/zh-cn/docs/contribute/blog/#contributor-blog)可能更为合适；  
+如果博客文章是关于开源的普遍话题，那么它可能更适合发布在 Kubernetes 项目之外的其他网站上。
+
+这种对目标受众的考量同样适用于原创文章和镜像文章。
+
+<!--
+The Kubernetes project is willing to mirror any blog article that was published to https://kubernetes.dev/blog/
+(the contributor blog), provided that all of the following criteria are met:
+
+- the mirrored article has the same publication date as the original (it should have the same publication time too,
+  but you can also set a time stamp up to 12 hours later for special cases)
+
+- For PRs that add a mirrored article to the main blog *after* the original article has merged into the contributor blog, ensure that all of the following criteria are met:
+    - No articles were published to the main blog after the original article was published to the contributor blog.
+    - There are no main blog articles scheduled for publication between the publication time of the original article and the publication time of your mirrored article.
+    
+  This is because the Kubernetes project doesn't want to add articles to people's feeds, such as RSS, except at the very end of their feed.
+-->
+Kubernetes 项目愿意镜像任何发布在 https://kubernetes.dev/blog/ （贡献者博客）上的文章，但前提是满足以下所有条件：
+
+- 镜像文章的发布日期必须与原始文章相同（发布时间也应相同，但在特殊情况下，可以设置最多延迟 12 小时的时间戳）。
+
+- 对于在原始文章已合并到贡献者博客**之后**，向主博客添加镜像文章的 PR，请确保满足以下所有条件：
```

Comment:
```suggestion
- 对于在原始文章已合并到贡献者博客*之后*，向主博客添加镜像文章的 PR，请确保满足以下所有条件：
```

Reply:
Since italics do not display well in Chinese, italics are usually expressed in bold when translating.

Reply:
thx